JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardYourNetwork.kt

```diff
@@ -48,43 +47,41 @@ import com.example.jetnews.ui.navigateTo
 fun PostCardPopular(post: Post, modifier: Modifier = Modifier) {
     Card(
         shape = MaterialTheme.shapes.medium,
-        modifier = modifier.preferredSize(280.dp, 240.dp)) {
-        Clickable(
-            modifier = Modifier.ripple(),
-            onClick = { navigateTo(Screen.Article(post.id)) }
-        ) {
-            Column {
-                val image = post.image ?: imageResource(R.drawable.placeholder_4_3)
-                Image(
-                    asset = image,
-                    contentScale = ContentScale.Crop,
-                    modifier = Modifier
-                        .preferredHeight(100.dp)
-                        .fillMaxSize())
+        modifier = modifier
+            .preferredSize(280.dp, 240.dp)
+            .clickable(onClick = { navigateTo(Screen.Article(post.id)) })
+    ) {
+        Column {
+            val image = post.image ?: imageResource(R.drawable.placeholder_4_3)
+            Image(
+                asset = image,
+                contentScale = ContentScale.Crop,
+                modifier = Modifier
+                    .preferredHeight(100.dp)
+                    .fillMaxSize())
 
-                Column(modifier = Modifier.padding(16.dp)) {
-                    val emphasisLevels = EmphasisAmbient.current
-                    ProvideEmphasis(emphasisLevels.high) {
-                        Text(
-                            text = post.title,
-                            style = MaterialTheme.typography.h6,
-                            maxLines = 2,
-                            overflow = TextOverflow.Ellipsis
-                        )
-                        Text(
-                            text = post.metadata.author.name,
-                            maxLines = 1,
-                            overflow = TextOverflow.Ellipsis,
-                            style = MaterialTheme.typography.body2
-                        )
-                    }
-                    ProvideEmphasis(emphasisLevels.high) {
-                        Text(
-                            text = "${post.metadata.date} - " +
-                                    "${post.metadata.readTimeMinutes} min read",
-                            style = MaterialTheme.typography.body2
-                        )
-                    }
+            Column(modifier = Modifier.padding(16.dp)) {
```

Comment:
Doesn't look like this column is needed?

Reply:
It's setting the padding all around those text components. It's much easier than adding different paddings to each.